server.py

```
@@ -19,240 +19,323 @@
         try:
-            return json.loads(content)
+            return response.json()
         except json.JSONDecodeError as e:
             logger.error(f"Failed to parse JSON response: {str(e)}")
-            raise Exception(f"Failed to parse Gerrit response as JSON: {str(e)}")
+            raise Exception(f"Failed to parse GitLab response as JSON: {str(e)}")
             
     except requests.exceptions.RequestException as e:
         logger.error(f"REST request failed: {str(e)}")
         if hasattr(e, 'response'):
             logger.error(f"Response status: {e.response.status_code}")
```

Ruff (0.15.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)